lib/organize.js

```diff
@@ -24,7 +31,13 @@ module.exports.organize = (
 	const languageService = getLanguageService(parser, filepath, code);
 
 	const fileChanges = languageService.organizeImports(
-		{ type: 'file', fileName: filepath, skipDestructiveCodeActions: organizeImportsSkipDestructiveCodeActions },
+		{
+			type: 'file',
+			fileName: filepath,
+			skipDestructiveCodeActions: organizeImportsSkipDestructiveCodeActions,
```

Comment:
```suggestion
```

I assume we can skip this now that we set `mode` based on it, since the property is deprecated anyway.